src/FormlyMapper/FormlyField/BirthdayField.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+use Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+class BirthdayField extends FormlyField
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFieldType()
+    {
+        return 'birthday';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function buildFieldTypeConfiguration(): void
+    {
+        $this->formlyFieldConfiguration['templateOptions']['type'] = $this->getFieldType();
+
+        $order = 'desc';
```

Comment:
This and 'asc' should be constants.

Reply:
This should be removed and set on the code suggestion below.

Reply:
done! thanks for the review!